python/packages/nisar/workflows/stage_watermask.py

```diff
@@ -108,7 +125,66 @@ def check_dateline(poly):
     return polys
 
 
-def determine_polygon(ref_slc, bbox=None):
+def adjust_lat_lon_coordinates(x_min, y_min, x_max, y_max, vrt_filename):
+    """Adjust a bbox to fall within the bounds of input Raster (vrt_filename)
+
+    Parameters
+    ----------
+    x_min: float
+        Minimum X coordinate of the bbox
+    y_min: float
+        Minimum Y coordinate of the bbox
+    x_max: float
+        Maximum X coordinate of the bbox
+    y_max: float
+        Maximum Y coordinate of the bbox
+    vrt_filename: str
+        The VRT filename of the input Raster (Water Mask)
+
+    Returns
+    -------
+    x_min: float
+        Adjusted minimum X coordinate of the bbox
+    y_min: float
+        Adjusted minimum Y coordinate of the bbox
+    x_max: float
+        Adjusted maximum X coordinate of the bbox
+    y_max: float
+        Adjusted maximum Y coordinate of the bbox
+    """
+    # the original COPERNICUS Water Mask and consequently the COPERNICUS Water Mask for NISAR
```

Comment:
```suggestion
    # the original Water Mask and consequently the Water Mask for NISAR
```
You probably want to remove the mentions to COPERNICUS?